src/cgpt/cgpt_common.c

```diff
@@ -647,6 +648,7 @@ static struct {
   {&guid_coreos_rootfs, "coreos-rootfs", "CoreOS rootfs"},
   {&guid_coreos_resize, "coreos-resize", "CoreOS auto-resize"},
   {&guid_coreos_reserved, "coreos-reserved", "CoreOS reserved"},
+  {&guid_coreos_root_raid, "coreos-root-raid", "CoreOS root on raid"},
```

Comment:
The docs have `CoreOS RAID containing root`, which is more correct.  Does that fit?  (or `RAID with root`)